### DISCUSSION TITLE
Powering PiJuice from a more powerful solar panel

### DISCUSSION THREAD

**Discussion by santisantis** [PiSupply/PiJuice#774] [Q&A]
Hi All, 
I have two questions. I'm currently using PiJuice Zero with a 40 W PiJuice solar panel.
1) Is it possible to power it up from a more powerful panel like 50 W (Sun Plus S 50)? Or will it require a step down converter, which cap the voltage at 10 V?
2) Is it possible to have two power supplies plugged into PiJuice + Raspberry setup? For instance having solar panel attached to the micro usb port of the PiJuice and have an regular charger plugged into Raspberry Pi. This would allow to have some back up power supply in case the it's cloudy etc.

I would be much obliged for any help provided.

**Comment by tvoverbeek** [comment]
1. The OiJuice charger control chip can only handle inputs 4.5 - 10 V. SO yes, you need to cap it at 10 V.
2. Yes, you can. You need to specify which input has precedence for charging the 5V from the RPi GPIO or the solar array input.

**Reply by santisantis** [reply]
Thank you very much for your answer.

**Reply by santisantis** [reply]
One more question, in this setup power coming from the solar panel should be connected to the PiJuice (mircro usb or +/- pin) and the regular charger plugged into the Raspberry Pi itself?

**Reply by tvoverbeek** [reply]
Yes, regular charger = RPi 5V power supply

**Reply by santisantis** [reply]
I'm currently powering PiJuice and Raspberry Pi from their microusbs using two 5V chargers. 
![image](https://user-images.githubusercontent.com/19371518/130960212-ad3078ff-b2a6-4b28-b3ab-a809aaa6d0c4.png)
In the PiJuice CLI, when enter into status section is see the following data. Both chargers are the same, but the GPIO (micro USB from Raspberry PI I presume) has a "BAD" status. Is this expected?

**Reply by tvoverbeek** [reply]
What is the setting of Inputs Precedence (General HAT config) on your PiJuice?
Anyway, 4.884V on the GPIO is too low. Should be close or a little over 5V.
